docs/_data/sidebars/_documentation.yml

```diff
@@ -72,6 +75,9 @@ entries:
               - title: Parametrize templates
                 url: /usage/deploy/values.html
 
+              - title: Debug templates
+                url: /usage/deploy/debug_templates.html
```

Comment:
Same here, there shouldn't be much content. Maybe bring it back to the Helm templating page?